exercice.js

```diff
@@ -0,0 +1,181 @@
+let chaussures = [
+  {
+    marque: 'Nike',
+    titre: 'Jordan Air',
+    collector: true,
+    taillesDispo: [28, 29, 30],
+    prix: 300,
+    elements: {
+      lacets: 'rouge',
+      corps: 'blanc',
+      semelle: 'bleu',
+    },
+  },
+  {
+    marque: 'Vans',
+    titre: 'Old Skool',
+    collector: false,
+    taillesDispo: [34, 36, 38],
+    prix: 70,
+    elements: {
+      lacets: 'noir',
+      corps: 'marron',
+    },
+  },
+  {
+    marque: 'Adidas',
+    titre: 'Stan Smith',
+    collector: false,
+    taillesDispo: [40, 42, 44],
+    prix: 100,
+    elements: {
+      lacets: 'vert',
+      corps: 'blanc',
+    },
+  },
+  {
+    marque: 'Nike',
+    titre: 'Air Force',
+    collector: true,
+    taillesDispo: [36, 37, 38],
+    prix: 200,
+    elements: {
+      lacets: 'bleu',
+      corps: 'marron',
+    },
+  },
+  {
+    marque: 'Nike',
+    titre: 'Air Max',
+    collector: true,
+    taillesDispo: [36, 38, 40],
+    prix: 900,
+    elements: {
+      lacets: 'jaune',
+      corps: 'vert',
+    },
+  },
+  {
+    marque: 'Vans',
+    titre: 'Era',
+    collector: false,
+    taillesDispo: [30, 32, 40],
+    prix: 40,
+    elements: {
+      lacets: 'bleu',
+      corps: 'noir',
+    },
+  },
+  {
+    marque: 'Vans',
+    titre: 'Sk8',
+    collector: false,
+    taillesDispo: [28, 30, 26],
+    prix: 40,
+    elements: {
+      lacets: 'bleu',
+      corps: 'noir',
+      languette: 'rouge',
+    },
+  },
+  {
+    marque: 'Vans',
+    titre: 'Old Skool',
+    collector: false,
+    taillesDispo: [36, 28, 60],
+    prix: 60,
+    elements: {
+      lacets: 'blanc',
+      corps: 'noir',
+      languette: 'noir',
+    },
+  },
+  {
+    marque: 'Nike',
+    titre: 'Nike SB',
+    collector: false,
+    taillesDispo: [30, 31, 32],
+    prix: 50,
+    elements: {
+      lacets: 'blanc',
+      corps: 'noir',
+    },
+  },
+  {
+    marque: 'Adidas',
+    titre: 'Old Skool',
+    collector: false,
+    taillesDispo: [29, 30, 32],
+    prix: 70,
+    elements: {
+      lacets: 'rouge',
+      corps: 'bleu',
+    },
+  },
+];
+
+// 1. Faire une boucle sur tout le tableau et de log les chaussures une par une
+
+for (var i = 0; i < chaussures.length; i++) {
```

Comment:
utilise les méthodes de tableaux 'forEach' 'filter' à la place de tes for stp (pour tous les points)